packages/components/select/demo/Loading.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <IxSpace vertical>
+    <IxSelect v-model:value="value" :dataSource="dataSource" :loading="isLoading"></IxSelect>
+    <IxButton @click="onLoad">Load Data</IxButton>
+  </IxSpace>
+</template>
+<script setup lang="ts">
+import { ref } from 'vue'
+
+import { SelectData } from '@idux/components/select'
+
+const dataSource = ref<SelectData[]>([{ label: 'Tom', value: 'tom' }])
+
+const value = ref('tom')
+const isLoading = ref(false)
+
+const timer = ref<number | null>(null)
+
+const onLoad = () => {
+  clear()
+
+  if (timer.value) {
+    clearTimeout(timer.value)
+  }
+  timer.value = setTimeout(() => {
+    isLoading.value = false
+    dataSource.value.push({ label: 'Jack', value: 'jack' })
+    dataSource.value.push({ label: 'Alan', value: 'alan' })
+  }, 3000)
+}
+
+function clear() {
+  isLoading.value = true
+  if (dataSource.value.length > 1) {
+    dataSource.value.length = 1
+  }
+}
```

Comment:
```suggestion
const dataSource = ref<SelectData[]>([{ label: 'Tom', key: 'tom' }])
const value = ref('tom')
const isLoading = ref(false)

const onLoad = () => {
  if (isLoading.value){
    return 
  }
  isLoading.value = true
  setTimeout(() => {
    isLoading.value = false
    dataSource.value = [{ label: 'Tom', key: 'tom' }, { label: 'Jack', key: 'jack' }, { label: 'Alan', key: 'alan' }]
  }, 3000)
}
```

对于响应式数组，尽可能使用重新赋值的方式去操作数据。